pages/contact.tsx

```diff
@@ -18,16 +18,17 @@ import { Box, Flex, ListItem, Text, UnorderedList } from "@chakra-ui/react";
 import React from "react";
 import Card from "../components/common/Card";
 import Icon from "../components/common/Icon";
-import Head from "next/head";
 import Breadcrumbs from "../components/common/Breadcrumbs";
+import MetadataTags from "../components/details/MetadataTags";
 
 export default function Contact() {
+  const pageTitle = "COKI: Contact";
+  const pageDescription = "Contact the Curtin Open Knowledge Initiative.";
```

Comment:
Could call variables like these title and description rather than pageTitle and pageDescription as they are shorter and there isn't really a need to differentiate the variables in this case.